### DISCUSSION TITLE
Apps installed with brew do not have autocomplete

### DISCUSSION THREAD

**Discussion by DavidConnack** [Homebrew/discussions#3322] [Linux] [answered]
I have installed brew on pop os.
apps installed fine and are working.
Auto complete does not work with any apps installed with brew.
Apps installed with apt have auto complete.
Any ideas?

**Comment by SMillerDev** [comment, marked as answer]
Have you followed https://docs.brew.sh/Shell-Completion ?

**Comment by DavidConnack** [comment]
Thank you so much.
I did see that, but I thought it was for auto complete for the brew app itself, not the apps that brew installs.
It worked perfectly.

**Reply by pedro-carmine** [reply]
Hi,
I'm having the same problem, followed the doc above and it only auto complete the brew app itself, it is not autocompleting pyenv, for instance.

Do you suggest anything?

edit: I am using macOS
